tools/Google.Cloud.Tools.VersionCompat/Program.cs

```diff
@@ -137,7 +154,7 @@ static int Main(string[] args)
 
             var diffs = Check(assem0, assem1, null);
 
-            if (diffs.Major.Any())
+            if ((options.OutputBehavior & OutputBehavior.Major) != 0 && diffs.Major.Any())
```

Comment:
Could use Enum.HasFlag, but that could come later.

Reply:
ack :) I'd forgotten that existed!